### PR TITLE
Reformat progs.pl

### PR DESCRIPTION
**Pull request by richsalz**
This reformats apps/progs.pl as done in the old GitLab MR 3303.

It also adds ARIA to the command table.  I assumed there's no modes, but ping @paulidale to confirm that (the second commit).

### CODE REVIEW

File: apps/progs.pl [hunk 1]
```diff
@@ -131,6 +131,7 @@
     "aes-128-cbc", "aes-128-ecb",
     "aes-192-cbc", "aes-192-ecb",
     "aes-256-cbc", "aes-256-ecb",
+    "aria",
```
Comment (levitte) [inline apps/progs.pl]: This doesn't seem to be enough, according to tests in Travis:

```
aria is not a known cipher
../util/shlib_wrap.sh ../apps/openssl aria -bufsize 113 -e -k test -in ./p -out ./p.aria.cipher => 1
not ok 15 - aria

#   Failed test 'aria'
#   at ../test/recipes/20-test_enc.t line 62.
aria is not a known cipher
../util/shlib_wrap.sh ../apps/openssl aria -bufsize 113 -a -e -k test -in ./p -out ./p.aria.cipher => 1
not ok 16 - aria base64

#   Failed test 'aria base64'
#   at ../test/recipes/20-test_enc.t line 62.
```

Comment (levitte) [inline apps/progs.pl]: Yup, like I though.  You have to use the shortnames that you find in `crypto/objects/objects.txt`

Comment (paulidale) [inline apps/progs.pl]: ARIA is only available via the EVP interface, not as a command name.
Modes are supported.

Comment (levitte) [inline apps/progs.pl]: Errrr...  the `enc` command uses the EVP interface, and what `progs.pl` does here is to produce aliases for algorithms that can be found with `EVP_get_cipherbyname()`.

Comment (paulidale) [inline apps/progs.pl]: My bad, it does work as a command name.
The mode needs to be included.

Comment (paulidale) [inline apps/progs.pl]: Looks like our messages got interleaved.  Thanks for the clarification.

Comment (richsalz) [inline apps/progs.pl]: additional commit pushed

**Comment by richsalz** [comment]
merged, thanks.

**Pull request closed without merging**